application/src/test/java/org/thingsboard/server/controller/AlarmControllerTest.java

```diff
@@ -568,10 +570,13 @@ public void testUnassignTenantUserAlarmOnUserRemoving() throws Exception {
         long beforeAssignmentTs = System.currentTimeMillis();
 
         doPost("/api/alarm/" + alarm.getId() + "/assign/" + savedUser.getId().getId()).andExpect(status().isOk());
-        AlarmInfo foundAlarm = doGet("/api/alarm/info/" + alarm.getId(), AlarmInfo.class);
-        Assert.assertNotNull(foundAlarm);
-        Assert.assertEquals(savedUser.getId(), foundAlarm.getAssigneeId());
-        Assert.assertTrue(foundAlarm.getAssignTs() >= beforeAssignmentTs);
+        Alarm finalAlarm = alarm;
+        var alarmObj = new Object() {
```

Comment:
I understand that this object is for lambda.
But it looks over-engineered.
I suggest marking it **final**
And call it some **wrapper** to describe the purpose of the object;

The simpler way to feed the lambdas - is a **single-sized array**.
``` java
final AlarmInfo[] foundAlarm = new AlarmInfo[1];
foundAlarm[0] = doGet () ...
```